PYXAID2/runMD2.py

```diff
@@ -469,15 +469,15 @@ def runMD(params):
 
 
     # nac_method = 0 : non spin polarized/SOC, multiple k-point
```

Comment:
Actually, lets also change the meaning of the options
0 - non-relativistic, non spin-polarized
1 - non-relativistic, spin-polarized
2 - relativistic, direct (only spin-adiabatic representation)
3 - relativistic, projections (spin-diabatic representation)